src/native_main.nim

```diff
@@ -73,6 +75,34 @@ proc findUserConfigFile(): Option[string] =
 
     return config_path
 
+proc removeFileAttribute(file, attr: string): int =
+    var xattrErr = 0
+    var xattrCmd = ""
+
+    if DEBUG:
```

Comment:
I think I'd prefer a `log()` function that contained the if to save us having them everywhere